src/components/specimen/components/contentBlock/SpecimenOverview.tsx

```diff
@@ -0,0 +1,155 @@
+/* Import Dependencies */
+import { Row, Col, Card } from 'react-bootstrap';
+
+/* Import Store */
+import { useAppSelector } from 'app/hooks';
+import { getSpecimen } from "redux/specimen/SpecimenSlice";
+
+/* Import Styles */
+import styles from 'components/specimen/specimen.module.scss';
+
+
+const SpecimenOverview = () => {
+    /* Base variables */
+    const specimen = useAppSelector(getSpecimen);
+
+    /* Check for Organization logo */
+    const logo = (ror: string) => {
+        try {
+            return require(`../../../../webroot/img/organizationLogo/${ror}.png`);
+        } catch (err) {
+            return null;
+        }
+    };
+
+    return (
+        <Row className="h-100">
+            <Col>
+                <Row className="h-50">
+                    {/* Location */}
+                    <Col md={{ span: 7 }} className="pb-2">
+                        <Card className="position-relative h-100">
+                            <Card.Body>
+                                <Card.Title>
+                                    Location
+                                </Card.Title>
+
+                                <Row className="mt-3">
+                                    <Col>
+                                        Country: Netherlands
```

Comment:
I think it is better to keep dummy values in the fields we haven't got harmonised data for. Just so it is very clear to anyone coming to the page that this still needs to be done. Maybe something like "In Progress" Or "Coming Soon" something to make clear that it is not actual data.
Say we ingest a bunch from Paris and the for some reason check our site we don't want them to think we messed up their data and it is now in the Netherlands or something

Reply:
Yes that sounds like something that could happen, changed all the dummy fields to: 'Coming Soon'.